auto_dev/data/repo/templates/autonomy/install.sh

```diff
@@ -1,19 +1,16 @@
-#!/usr/bin/env bash
-
-set -euo pipefail
```

Comment:
this forces the script to fail if any of the commands fail.
Please dont remove it!

Reply:
reverted here 79afc73e5c86ed64a8afa8c4da1870b7ba80c036